File: fonte/api/src/main/java/com/AchadosPerdidos/API/Domain/Entity/Role.java
package com.AchadosPerdidos.API.Domain.Entity;

import lombok.Getter;
import lombok.Setter;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;
import java.util.Date;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Role {
    private Integer id;
    private String nome;
    private String descricao;
    private Date dtaCriacao;
    private Boolean flgInativo;
    private Date dtaRemocao;
}

File: flutter/lib/domain/entities/role.dart
class Role {
  final int id;
  final String nome;
  final String? descricao;
  final DateTime? dtaCriacao;
  final bool flgInativo;
  final DateTime? dtaRemocao;

  Role({
    required this.id,
    required this.nome,
    this.descricao,
    this.dtaCriacao,
    this.flgInativo = false,
    this.dtaRemocao,
  });
}

